.jules/sentinel.md

```
@@ -0,0 +1,10 @@
+## 2024-04-19 - Fix SQL Injection in DemographicDaoImpl Native Queries ORDER BY
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)